webview/src/plots/components/comparisonTable/cell/ComparisonTableMultiCell.tsx

```diff
@@ -24,7 +24,16 @@ export const ComparisonTableMultiCell: React.FC<{
     <div data-testid="multi-image-cell" className={styles.multiImageWrapper}>
       <ComparisonTableCell
         path={path}
-        plot={{ id: plot.id, imgs: [plot.imgs[currentStep]] }}
+        plot={{
+          id: plot.id,
+          imgs: [
+            plot.imgs[currentStep] || {
```

Comment:
Running experiments start off with the workspace's images than go on to hold their own created images (Is that normal or is there a bug with `plot diff`?). This means that an revision could start off with 20 images in the beginning of an experiment run if the `workspace` originally had 20 than less than 20 by the end. 

A user could set a multi image slider to a step that doesn't actually exist within the experiment during the beginning of the experiment run. This would cause the plots to crash once the images get updates to a smaller length. This change handles this scenario and stops plots from crashing.